src/validation/schema.js

```diff
@@ -322,13 +378,200 @@ const validateEntityFields = (defs, def) =>
     List(),
   )
 
+const validateNoImportsDirective = def =>
+  def.directives.find(directive => directive.name.value == 'imports')
+    ? List().push(
+        immutable.fromJS({
+          loc: def.name.loc,
+          entity: def.name.value,
+          message: `@imports directive only allowed on '${SUBGRAPH_SCHEMA}' type`,
+        }),
+      )
+    : List()
+
+const importDirectiveTypeValidators = {
+  StringValue: (_def, _directive, _type) => List(),
```

Comment:
Is `StringValue` a thing? Isn't t just `String`?

Reply:
It is indeed `StringValue`

Reply:
Ah, of course it is. Sorry. 😄